package-configuration/linux-cli/get-cloudify.py

```diff
@@ -332,8 +369,9 @@ def parse_args(args=None):
             help='Python path to use (defaults to "python").')
     else:
         parser.add_argument(
-            '--nosudo', action='store_true',
-            help='Do not use sudo for prerequisites.')
+            '--nodrop', action='store_true',
```

Comment:
Not everyone would want to drop the privileges. I would be funny to have a user run something with sudo but not provide an option for now creating the virtualenv and installing the cli as a sudoer.